### PR TITLE
Support vpa

### PR DESCRIPTION
**Pull request by jiangpengcheng**
resolve: #541

### CODE REVIEW

File: src/main/java/io/functionmesh/compute/util/FunctionsUtil.java [hunk 1]
```diff
@@ -401,6 +412,77 @@ public static V1alpha1Function createV1alpha1FunctionFromFunctionConfig(String k
         }
 
         V1alpha1FunctionSpecPod specPod = new V1alpha1FunctionSpecPod();
+
+        if (customRuntimeOptions.getVpaSpec() != null ) {
```
Comment (freeznet) [inline src/main/java/io/functionmesh/compute/util/FunctionsUtil.java:416]: could you please introduce a new function that convert `VPASpec` to `V1alpha1FunctionSpecPodVpa` so we can reduce the complexity of `createV1alpha1FunctionFromFunctionConfig`?

**Review by freeznet — APPROVED** [review]

File: src/main/java/io/functionmesh/compute/models/CustomRuntimeOptions.java [hunk 2]
```diff
@@ -29,6 +29,7 @@ public class CustomRuntimeOptions {
     private String outputTypeClassName;
     private int maxReplicas;
     private Map<String, String> env;
+    private VPASpec vpaSpec;
```
Comment (freeznet) [inline src/main/java/io/functionmesh/compute/models/CustomRuntimeOptions.java:32]: this definition should keep similarity as the https://github.com/streamnative/function-mesh-worker-service/pull/554 did, could you please work with @tpiperatgod to make sure we have similar naming for both VPA and HPA? thanks.

**Review by freeznet — DISMISSED** [review]

File: src/main/java/io/functionmesh/compute/util/FunctionsUtil.java (hunk 1 above)
Comment (jiangpengcheng) [reply src/main/java/io/functionmesh/compute/util/FunctionsUtil.java:416]: good point

File: src/main/java/io/functionmesh/compute/models/CustomRuntimeOptions.java (hunk 2 above)
Comment (jiangpengcheng) [reply src/main/java/io/functionmesh/compute/models/CustomRuntimeOptions.java:32]: ok

**Review by freeznet — APPROVED** [review]

**Pull request merged**